### PR TITLE
Dbaas 2571

### PR DESCRIPTION
**Pull request by Ben-Epstein**
New PySplice code for MLManager work in K8s as well as some updated functions

### CODE REVIEW

File: splicemachine/ml/management.py
Comment (edriggers) [inline splicemachine/ml/management.py:78]: Looks like you are trying to parse a json object that is represented as a string.  Is there a more native way to do this?

File: splicemachine/ml/management.py
Comment (edriggers) [inline splicemachine/ml/management.py:121]: I believe for performance you want this to be a parameterized statement so that the statement is cached.  select "binary" from artifacts where name = ? and run_uuid = ?

File: splicemachine/ml/management.py
Comment (edriggers) [inline splicemachine/ml/management.py:139]: Do we want to use logging?

File: splicemachine/ml/management.py
Comment (edriggers) [inline splicemachine/ml/management.py:243]: This is a comment overall of this file.  Do we want to use logging instead of printing to the console.

**Review by edriggers — APPROVED** [review]
Approving with the caveat that I am not that familiar with the intent of this PR.  There are a couple of minor questions but nothing that should block merging this PR.

File: splicemachine/ml/management.py
Comment (Ben-Epstein) [reply splicemachine/ml/management.py:139]: We do generally want to use logging, and I can add a log here, but we also want to print here so the user knows where it's being tracked.

File: splicemachine/ml/management.py
Comment (Ben-Epstein) [reply splicemachine/ml/management.py:243]: In some cases, yes. But many of these prints are for the user as well. But I should go back and add logging. Maybe after merge, make a new Jira. Or before merge, worth discussing.

**Review by PatrickSplice — APPROVED** [review]

**Review by njnygaard — APPROVED** [review]

**Pull request merged**
